code/src/main/java/habit_mode/model/SuccessCodes.java

```diff
@@ -0,0 +1,50 @@
+package habit_mode.model;
+
+/**
+ * The enum SuccessCodes.
+ * 
+ * @author Team 1
+ * @version Spring 2022
+ */
+public enum SuccessCodes {
```

Comment:
Enums in Java should be singular. `SuccessCodes` should be `SuccessCode`, as an enum variable is referring to a single code at any given time.

Reply:
Subsequent commits have fixed this issue.